src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/NavigationTests.java

```diff
@@ -76,4 +76,26 @@ public void MercuryCuratedNavigationTests_002_navigateThroughSection() {
 
     UrlChecker.isPathContainedInCurrentUrl(driver, ROOT_PATH_SECTION + section.getTitle());
   }
+
+  // CCT09
+  @Test(groups = {"MercuryCuratedNavigationTests_003", "MercuryCuratedNavigationTests",
+                  "MercuryCuratedContentTests", "Mercury"})
+  public void MercuryCuratedNavigationTests_003_navigateThroughDifferentUrl() {
+    CuratedContentPageObject section = new CuratedContentPageObject(driver);
+
+    String expectedUrl = wikiURL + MercurySubpages.CC_CATEGORY_TEMPLATES;
+    section.navigateToUrlWithPath(wikiURL, MercurySubpages.CC_CATEGORY_TEMPLATES);
```

Comment:
we must discuss how we match the two navigation concepts of tests:
1. navigating through url's using get method
2. navigating through url's using click actions that trigger and return other Web(Page)Objects